static/js/shortcuts.js

```diff
@@ -1,5 +1,11 @@
 // Press arrow keys to turn a page
 (() => {
+  // Make code blocks focusable, so we can use activeElement to check if user is scrolling inside a code block
+  const preElements = document.querySelectorAll("pre");
+  preElements.forEach((element) => {
```

Comment:
这个 `preElements` 常量创建之后下面好像没有再用到它，所以好像也不必新建这么个常量吧：

```suggestion
  document.querySelectorAll("pre").forEach((element) => {
```